subsys/debug/Kconfig

```diff
@@ -54,6 +55,15 @@ config ASAN
 	  compiler with the ``-fsanitize=address`` command line option, and
 	  the libasan library.
 
+config UBSAN
+	bool "Build with undefined behavior sanitizer"
+	depends on ARCH_POSIX
+	help
+	  Builds Zephyr with Undefined Behavior Sanitizer enabled.
+	  This is currently only supported by the native_posix port, and requires
```

Comment:
Just a minor nit:
It is supported by all posix arch boards, not just native_posix. could you rephrase
"This is currently only supported by the native_posix port, and requires"
to
"This is only supported by boards based on the posix arch, and requires"

Reply:
Updated description and took the freedom to lineup the ASAN description with the one used for UBSAN